themes/frontierline/content-views/content-page-sidebar.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Display a full post with full size image.
+ */
+?>
+
+<?php if (has_post_thumbnail()) : ?>
+  <div class="post-image post-image-full">
+    <?php the_post_thumbnail('post-full-size'); ?>
+  </div>
+<?php endif; ?>
+
+<header class="entry-header">
+  <div class="entry-tools">
+  <?php if (has_category()) : ?>
```

Comment:
Since this is explicitly a static page template we can probably eliminate categories as they'll never appear.